src/qml/filters/stabilize/ui.qml

```diff
@@ -29,19 +29,36 @@ Item {
     property string settingsSavePath: settings.savePath
 
     function setStatus( inProgress ) {
```

Comment:
@bmatherly  I fixed the if() whitespace by altering my commit and remove the whitespace in setStatus() with a separate commit.